Issue and PR-Template (#428)
OpenEnergyPlatform/open-MaStR: .github/pull_request_template.md

## Summary of the discussion

Describe the findings of the discussion in the issue or meeting.

## Type of change (CHANGELOG.md)

### Added

> - Add a new class [(#)](https://github.com/rl-institut/super-repo/pull/)

Reviewer: ```suggestion
- Add a new class [(#)](https://github.com/OpenEnergyPlatform/open-MaStR/pull/)
```

### Updated

> - Update a definition [(#)](https://github.com/rl-institut/super-repo/pull/)

Reviewer: ```suggestion
- Update a definition [(#)](https://github.com/OpenEnergyPlatform/open-MaStR/pull/)
```

### Removed

> - Remove a broken link [(#)](https://github.com/rl-institut/super-repo/pull/)

Reviewer: ```suggestion
- Remove a broken link [(#)](https://github.com/OpenEnergyPlatform/open-MaStR/pull/)
```

## Workflow checklist

### Automation
Closes #

### PR-Assignee

> - [ ] 🐙 Follow the workflow in [CONTRIBUTING.md](https://github.com/rl-institut/super-repo/blob/develop/CONTRIBUTING.md)

Reviewer: ```suggestion
- [ ] 🐙 Follow the workflow in [CONTRIBUTING.md](https://github.com/OpenEnergyPlatform/open-MaStR/blob/production/CONTRIBUTING.md)
```

> - [ ] 📝 Update the [CHANGELOG.md](https://github.com/rl-institut/super-repo/blob/develop/CHANGELOG.md)

Reviewer: ```suggestion
- [ ] 📝 Update the [CHANGELOG.md](https://github.com/OpenEnergyPlatform/open-MaStR/blob/production/CHANGELOG.md)
```

- [ ] 📙 Update the documentation

### Reviewer

> - [ ] 🐙 Follow the [Reviewer Guidelines](https://github.com/rl-institut/super-repo/blob/develop/CONTRIBUTING.md#40-let-someone-else-review-your-pr)

Reviewer: @steull please add the "Reviewer Guidlines" to this repo & update the link or delete the line

> - [ ] 🐙 Follow the [Reviewer Guidelines](https://github.com/rl-institut/super-repo/blob/develop/CONTRIBUTING.md#40-let-someone-else-review-your-pr)

Reviewer: ```suggestion
- [ ] 🐙 Follow the [Reviewer Guidelines](https://github.com/OpenEnergyPlatform/open-MaStR/blob/production/CONTRIBUTING.md#40-let-someone-else-review-your-pr)
```
